base/uk.ac.stfc.isis.ibex.opis/resources/SKFG5Chopper/skf_g5_chopper.py

```diff
@@ -0,0 +1,27 @@
+from org.csstudio.opibuilder.scriptUtil import PVUtil
+from org.csstudio.opibuilder.scriptUtil import ConsoleUtil
+
+
+container = display.getWidget("Tabs")
+
+# enable tabs
+tab_count = int(container.getPropertyValue("tab_count"))
+for i in range(tab_count):
+
+	if pvArray[i + 1].isConnected():
+		container.setPropertyValue("tab_%s_enabled" % i, "true")
+	else:
+		container.setPropertyValue("tab_%s_enabled" % i, "false")
+
+# activate asked for tab
+try:
+    active_tab = int(pvs[0].getValue().value)
+except (IndexError, AttributeError, TypeError):
+    active_tab = 1
+    ConsoleUtil.writeInfo('Could not get value from macro for active tab')
+
+if active_tab < 1 or active_tab > tab_count:
+    ConsoleUtil.writeInfo('Active tab is out of range was {}'.format(active_tab))
```

Comment:
Unfortunately BOY uses an older version of python, str.format() doesn't work.

Reply:
I tested this and it works on mine?? I will use older str format